includes/wp-admin/class-welcome-fields.php

```diff
@@ -19,6 +19,9 @@ class Welcome_Fields {
 	public static function init() {
 		\add_action( 'load-settings_page_activitypub', array( self::class, 'register_welcome_fields' ) );
 		\add_action( 'admin_print_styles-settings_page_activitypub', array( self::class, 'enqueue_styles' ) );
+
+		\add_action( 'added_option', array( self::class, 'resolve_checklist' ) );
+		\add_action( 'updated_option', array( self::class, 'resolve_checklist' ) );
```

Comment:
Hm, these would run for _every_ option addition or upgrade.

What did you not like about the option-specific versions?

Reply:
it's quite noise and there is a high change that you miss one or have a typo.

> Hm, these would run for every option addition or upgrade.

but that is a valid point.

Reply:
Let's revert and merge? should be ready for final testing then